src/twisted/trial/test/test_loader.py

```diff
@@ -66,6 +73,30 @@ def test_findFileWithImportError(self):
         )
 
 
+def loosely_resembles(module: ModuleType) -> Matcher[ModuleType]:
```

Comment:
naming convention nit: should be `looselyResembles`. If you want to go for consistency with hamcrest though, feel free to leave it alone

Reply:
Should this go into `twisted.trial.test.matchers` too?

Reply:
Oops, that was just force of habit.  I assume Twisted still requires camelCase even for something like "a function used with a library that uses underscores".

I put this one in the test module itself rather than the matchers module because it seems unlikely this will ever be useful for tests elsewhere - so I'd probably lean towards leaving it here.  Of course, someone writing tests elsewhere is not very likely to ever find this for re-use if it is in the test module so that might be an argument for having it in `matchers` instead...